packages/strapi-admin/admin/src/containers/App/index.js

```diff
@@ -75,11 +76,14 @@ function App(props) {
 
         if (uuid) {
           try {
+            const deviceId = await getUID();
```

Comment:
Don't you need this identifier in all request?

You might need to add it [here](https://github.com/strapi/strapi/blob/ebcfa07c338f3d1ac7fe29162fd9ab8edb4fba39/packages/strapi-admin/admin/src/containers/Admin/index.js#L94-L112)